src/contracts/tokens/biosample-permission-token.sol

```diff
@@ -65,4 +66,73 @@ contract BiosamplePermissionToken is
     emit URI(_uri, _tokenId);
   }
 
+  
+  /// TODO: add description
+  function createWithSignature(
+    uint256 _tokenId,
+    string calldata _permission,
+    bytes32 _r,
+    bytes32 _s,
+    uint8 _v
+  )
+    external
+  {
+    bytes32 _claim = getCreateClaim(_tokenId, _permission);
+    require(
+      isValidSignature(
+        address(_tokenId),
+        _claim,
+        _r,
+        _s,
+        _v
+      ),
+      "Signature is not valid."
+    );
+    NFToken._mint(address(_tokenId), _tokenId);
```

Comment:
validate authorized per tokenid namespacing

Reply:
That is automatically check with the signature check.